testcharms/charm-repo/kubernetes/storage-filesystem/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+name: storage-filesystem
```

Comment:
Do we need this storage charm fort this PR?

Reply:
hi @wallyworld yes, because some tests require k8s `storage-filesystem` charm.

https://pastebin.ubuntu.com/p/QZjjgVp3g7/